newswires/client/src/Item.tsx

```diff
@@ -79,10 +81,18 @@ export const Item = ({ id }: { id: string }) => {
 							<h2 id={pushedFlyoutTitleId}>{itemData.content.headline}</h2>
 						</EuiTitle>
 						<EuiSpacer size="xs" />
-						<WireDetail wire={itemData} />
+						<WireDetail wire={itemData} showJson={showJson} />
 					</EuiFlyoutBody>
-					<EuiFlyoutFooter>
+					<EuiFlyoutFooter
+						css={css`
+							display: flex;
+							justify-content: space-between;
+						`}
+					>
 						<EuiButton onClick={() => handleDeselectItem()}>Close</EuiButton>
+						<EuiButton onClick={() => setShowJson(!showJson)}>
+							Show JSON
```

Comment:
nice to have...
```suggestion
							{shouldShowJson ? "Hide" : "Show"} JSON
```

Reply:
unless EUI has a toggle button, in which case button text could be static

Reply:
ooh yeah toggle is a good shout! 
<img width="171" alt="image" src="https://github.com/user-attachments/assets/d9716a0f-50b5-4392-ac89-9e71b6df6848">